collectors/osd.go

```diff
@@ -0,0 +1,518 @@
+package collectors
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+//OsdCollector sample comment
+type OsdCollector struct {
+	conn Conn
+
+	//CrushWeight is a persistent setting, and it affects how CRUSH assigns data to OSDs.
+	//CrushWeight displays the CRUSH weight for the OSD
```

Comment:
Not a blocker, but the second line can be replaced by just `It displays ...`. As long the first word in the comment block matches with the variable or function name, it should be applied correctly.